Tests/XCUITests/BaseTestCase.swift

```diff
@@ -79,6 +79,15 @@ class BaseTestCase: XCTestCase {
         app.activate()
     }
 
+    func restartInBackground() {
+        // Send app to background, and re-enter
+        XCUIDevice.shared.press(.home)
+        // Let's be sure the app is backgrounded
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
```

Comment:
Nit: We could declare this at the top in case it is needed for other methods in the future